src/components/LandingInfo.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { Grid, Button } from '@material-ui/core'
+import { Link as RouterLink } from 'react-router-dom'
+
+export default class LandingInfo extends React.Component {
+  render() {
+    const xsButton = 8
+    const smButton = 3
+    return (
+      <div id="info">
+        <h1 id="info-title">What is ResumeBook?</h1>
+        <Grid container id="info-content">
+          <Grid item sm={12 - 1 - smButton} xs={12}>
```

Comment:
```suggestion
          <Grid item xs={12} sm={8} >
```
Using constants versus variables here for breakpoints makes it a lot more readable